packages/spacecat-shared-data-access/src/models/importer/import-job.js

```diff
@@ -89,6 +90,22 @@ const ImportJob = (data) => {
     return self;
   };
 
+  /**
+   * Updates the Url count of the ImportJob
+   * @param {number} urlCount - The new url count.
+   * @returns {ImportJob} The updated ImportJob object.
+   */
+  self.updateUrlCount = (urlCount) => {
+    if (!isNumber(urlCount)) {
```

Comment:
possible you were looking for `isInteger` versus `isNumber`

Reply:
Done!